tests/clients/test_object.py

```diff
@@ -263,7 +293,7 @@ def test_object_client_copy_multiple(mocker):
     )
```

Comment:
Maybe change the name of the above test to `test_object_client_copy_recursive`, or merge it and `test_object_client_copy_default` into a single parameterised test?